apps/docs/content/docs/sync.mdx

```diff
@@ -141,67 +103,15 @@ function registerUrlHandler(editor: Editor) {
 
 And [here's a full working example](https://github.com/tldraw/tldraw/blob/main/apps/simple-server-example/src/client/App.tsx) of the client-side code.
 
-#### WebSocket server
-
-The `@tldraw/sync-core` package exports a class called [`TLSocketRoom`](?) that should be created server-side on a per-document basis.
-
-`TLSocketRoom` is used to
-
-- Store an authoritative in-memory copy of the document state
-- Transparently set up communication between multiple sync clients via WebSockets.
-- Provide hooks for persisting the document state when it changes.
-
-<Callout icon="💡">
-	You should make sure that there's only ever one `TLSocketRoom` globally for each room in your app.
-	If there's more than one, users won't see each other and will overwrite others' changes. We use
-	[Durable Objects](https://developers.cloudflare.com/durable-objects/) to achieve this on
-	tldraw.com.
-</Callout>
-
-Read the reference docs for [`TLSocketRoom`](?), and see an example of how to use it in the [simple server example](https://github.com/tldraw/tldraw/blob/main/apps/simple-server-example/src/server/rooms.ts).
-
-#### Asset storage
+## Using tlsync
 
-As well as synchronizing the rapidly-changing document data, tldraw also needs a way to store and
-retrieve large binary assets like images or videos.
+## Custom shapes & bindings
```

Comment:
it feels really weird to introduce custom shapes & bindings before we've even gone over the other server setup parts. We make reference to `TLSocketRoom` in this section, but don't introduce that until the very bottom of the page.